pypxe-server.py

```diff
@@ -85,9 +89,22 @@
             dargs.update(loadedcfg)
             args = argparse.Namespace(**dargs)
 
+        # setup logger
+        logger = logging.getLogger(sys.argv[0])
```

Comment:
Why are you using `sys.argv[0]` here? Should you be using `argparse` like everywhere else? I could be misunderstanding this line but please clarify. Thanks.